.npmignore

```diff
@@ -1,15 +1,13 @@
 *
+packages/dd-trace/src/external-logger/test/index.spec.js
 !ext/**/*
 !packages/*/lib/**/*
 !packages/*/src/**/*
 !packages/*/index.js
-!scripts/**/*
+!scripts/preinstall.js
 !vendor/**/*
-!CONTRIBUTING.md
 !LICENSE
 !LICENSE-3rdparty.csv
-!MIGRATING.md
-!README.md
```

Comment:
We need the readme in the package for https://www.npmjs.com/package/dd-trace to display something useful.

Reply:
Good point